hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcClient.java

```diff
@@ -45,24 +46,45 @@
 @InterfaceAudience.LimitedPrivate(HBaseInterfaceAudience.CONFIG)
 public class NettyRpcClient extends AbstractRpcClient<NettyRpcConnection> {
 
+  protected final WriteBufferWaterMark writeBufferWaterMark;
+
+  protected static final String CLIENT_CONNECT_MAX_RETRIES = "hbase.ipc.client.connect.max.retries";
```

Comment:
These first three are unused. Any anyway, the parent class manages reading these values from configuration.

Reply:
Copy paste error, should just be WATERMARK stuff.